.github/workflows/push.yml

```diff
@@ -144,3 +144,11 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
           provenance: false
+      - id: tag-version
+        env:
+          IMAGE_TAG: ${{ steps.push.outputs.image_name }}.${{ steps.push.outputs.image_tag }}
```

Comment:
change this to 
[image_and_tag](https://github.com/entur/gha-docker/blob/main/README-push.md#output_image_and_tag)